Makefile

```
@@ -937,6 +937,7 @@ test/pel-mark-test.el.test-passed:              pel-mark.elc pel--base.elc
 test/pel-package-test.el.test-passed:           pel-package.elc pel--base.elc pel--options.elc
 test/pel-prompt-test.el.test-passed:            pel-prompt.elc
 test/pel-read-test.el.test-passed:              pel-read.elc
+test/pel-rst-test.el.test-passed:               pel-rst.elc
```

checkmake (0.3.2)

[warning] Target "test/pel-rst-test.el.test-passed" should be declared PHONY.

(phonydeclared)